MetagenomeUtils.spec

```diff
@@ -12,7 +12,42 @@ module MetagenomeUtils {
         - Other functions for BinnedContig manipulation, e.g. edit_bins, merge_bins ...
     */
 
+    /* 
+      An X/Y/Z style reference
+    */
+    typedef string obj_ref;
 
+    /*
+      file_directory: file directory containing compressed/unpacked contig file(s) to build BinnedContig object
+      assembly_ref: Metagenome assembly object reference
+      binned_contig_name: BinnedContig object name
+      workspace_name: the name/id of the workspace it gets saved to
+    */
+    typedef structure {
+      string file_directory;
+      obj_ref assembly_ref;
+      string binned_contig_name;
+      string workspace_name;
+    } FileToBinnedContigParams;
 
+    typedef structure {
+      obj_ref binned_contig_obj_ref;
+    } FileToBinnedContigResult;
+
+    /*
+      file_to_binned_contigs: Generating BinnedContigs ojbect from files
+
+      input params:
+      file_directory: file directory containing compressed/unpacked contig file(s) to build BinnedContig object
```

Comment:
Is this file_directory generic, or specific to MaxBin?  If it's just a list of contig files, then this is fine.  If it is specific to MaxBin, we should either rename the method to reflect that, or add any MaxBin specific logic to the MaxBin module.